eng/Versions.props

```diff
@@ -212,27 +212,27 @@
     <RoslynMicrosoftVisualStudioExtensionManagerVersion>0.0.4</RoslynMicrosoftVisualStudioExtensionManagerVersion>
     <SourceBrowserVersion>1.0.21</SourceBrowserVersion>
     <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
-    <SystemCompositionVersion>1.0.31</SystemCompositionVersion>
+    <SystemCompositionVersion>6.0.0</SystemCompositionVersion>
```

Comment:
that's quite the jump.  any concerns here? :)

Reply:
VS repo uses 6.0.0 as well.

Reply:
So apparently there is a concern: https://github.com/dotnet/roslyn/issues/59128 :(